src/MLJClusteringInterface.jl

```diff
@@ -614,4 +678,73 @@ report(mach).cutter(h = 2.5)
 """
 HierarchicalClustering
 
+"""
+$(MMI.doc_header(AffinityPropagation))
+
+[Affinity Propagation](https://en.wikipedia.org/wiki/Affinity_propagation) is a clustering algorithm based on the concept of "message passing" between data points. More information is available at the [Clustering.jl documentation](https://juliastats.org/Clustering.jl/stable/index.html). Use `predict` to get cluster assignments. Indices of the exemplars, their values, etc, are accessed from the machine report (see below).
+
+This is a static implementation, i.e., it does not generalize to new data instances, and
+there is no training data. For clusterers that do generalize, see [`KMeans`](@ref) or
+[`KMedoids`](@ref).
+
+In MLJ or MLJBase, create a machine with
+
+    mach = machine(model)
+
+# Hyper-parameters
+
+- `damp = 0.5`: damping factor
+
+- `maxiter = 200`: maximum number of iteration
+
+- `tol = 1e-6`: tolerance for converenge
+
+- `preference = nothing`: the value of the diagonal elements of the similarity matrix
```

Comment:
I guess this is correct, yes?

```suggestion
- `preference = nothing`: the value (single float) of the diagonal elements of the similarity matrix
```

Also, could you please document how an unspecified `preference` is handled. A reference to the wiki article is fine.